packages/stark-ui/src/modules/date-picker/components/date-picker.component.ts

```diff
@@ -92,11 +191,34 @@ export class StarkDatePickerComponent extends AbstractStarkUiComponent implement
 	@Input()
 	public pickerName: string = "";
 
+	/**
+	 * If the dropdown is required or not. by default, the dropdown is not required
+	 */
+	@Input()
+	public get required(): boolean {
+		return this._required;
+	}
+	public set required(value: boolean) {
+		const newValue: boolean = coerceBooleanProperty(value);
+
+		if (this._required !== newValue) {
```

Comment:
I think this if-block is not necessary and you can just directly set `required` after coercing it to a boolean.